client/landing/stepper/declarative-flow/internals/steps.tsx

```diff
@@ -211,6 +211,11 @@ export const STEPS = {
 		asyncComponent: () => import( './steps-repository/site-migration-instructions' ),
 	},
 
+	SITE_MIGRATION_CAPTURE: {
```

Comment:
Should this name be more consistent with the slug? Maybe `SITE_MIGRATION_IDENTIFY`?

Reply:
Fixed here https://github.com/Automattic/wp-calypso/pull/87999/commits/b803f25b04c0fd5afbbfb5b69be58adbc46f6511